src/widgets/dropdown/widget-dropdown-view.js

```diff
@@ -93,19 +88,19 @@ module.exports = cdb.core.View.extend({
   _togglePinned: function () {
     var pinned = !this.model.get('pinned');
```

Comment:
it's being done (check _initBinds ` this.model.bind('change:pinned change:collapsed change:normalized', this.render, this`)